app/templates/events/view/scheduler.hbs

```diff
@@ -15,7 +15,10 @@
         {{/each}}
       </div>
     </div>
-    <div class="thirteen wide column">
+    <div class="sixteen wide mobile thirteen wide computer column">
+      {{#if device.isMobile}}
+        <div class="ui hidden divider"></div>
```

Comment:
This can be 

```html
<div class="mobile only ui hidden divider"></div>
```

@kushthedude I'm not sure why you prefer to use `device.*` properties when there are other alternatives available ? DOM modifications are more costly than CSS media queries.

Reply:
`DOM modifications are more costly than CSS media queries.`.
Thanks for briefing , TBH , I was not knowing about the query `Mobile Only`

Reply:
@niranjan94 the following line is not working ?